acspy/acsc.py

```diff
@@ -192,6 +192,11 @@ def setJerk(hcomm, axis: int, jerk: double, wait=SYNCHRONOUS):
     call_acsc(acs.acsc_SetJerk, hcomm, axis, double(jerk), wait)
 
 
+def setRPosition(hcomm, axis: int, rpos: double, wait=SYNCHRONOUS):
```

Comment:
The type hints for these should be Python types, since we convert to ctypes internally

```suggestion
def setRPosition(hcomm, axis: int, rpos: float, wait=SYNCHRONOUS):
```